sdk/web-pubsub/arm-webpubsub/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## 0.1.0-beta.1 (2021-07-27)
+
+This is the first preview for the new version of the `@azure/arm-webpubsub` package that follows the new [guidelines for TypeScript SDKs](https://azure.github.io/azure-sdk/typescript_introduction.html) for Azure services.
```

Comment:
I think the changelog should follow the standard format where it has some standard subsections: Features added, Key Bug Fixes, etc

Reply:
@deyaaeldeen  Could you show me where's the standard format ? like an example?

Reply:
See subsections under https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/ai-text-analytics/CHANGELOG.md#520-beta1-unreleased